_docs/sysadmin/system_customization.md

```diff
@@ -166,6 +168,32 @@ However, this should be only applied to the `php.ini` running the web server and
 
 <small>[Back To Table of Contents](#table-of-contents)</small>
 
+### Increasing the max number of files that can be uploaded at once
+
+By default PHP restricts uploading 20 files at a time. To change this limit, edit:
+
+```
+/etc/php/7.2/fpm/php.ini
+```
+
+and modify the variable:
+
+```
+upload_max_filesize
+```
+
+Then restart PHP and apache
+```
+sudo systemctl restart apache2.service
+sudo systemctl restart php7.2-fpm.service
+```
+
+and/or
+
+```
+sudo service apache2 restart
+sudo service php7.2-fpm restart
+```
```

Comment:
remove this second bit and just leave the `systemctl` instructions. All targetted (and planned) distros have systemd and is the recommended way to manage these things.